plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -306,7 +307,7 @@ public boolean configure(String name, Map<String, Object> params) throws Configu
             _storage.configure("StorageLayer", params);
         }
 
-        value = _configDao.getValue(Config.VmwareCreateFullClone.key());
```

Comment:
I think this is the only usage that makes the configuration parameter not dynamic. Can we not use any indirect variables and make the parameter dynamic @Pearl1594 ?